SDK/include/Server/Components/Pawn/pawn.hpp

```diff
@@ -57,6 +58,7 @@ struct PawnLookup
 	IMenusComponent* menus = nullptr;
 	IObjectsComponent* objects = nullptr;
 	IPickupsComponent* pickups = nullptr;
+	IRecordingsComponent* recordings = nullptr;
```

Comment:
I think this is an ABI break, but the whole lookup system is bad and I should change it (especially since we already have a robust flexible component lookup system).

Reply:
Any workaround now or is this PR still blocked because of this?

Reply:
As pointed out on discord, this isn't a problem.